underscore.js

```diff
@@ -1100,6 +1100,9 @@
 
    // Return a copy of the object without the blacklisted properties.
   _.omit = restArgs(function(obj, keys) {
+    if(obj != null && Object.getOwnPropertySymbols(obj) != null){
```

Comment:
`Object.getOwnPropertySymbols` is a relatively recent addition. IE doesn't support it yet, for instance. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertySymbols#Browser_compatibility